website/static/js/newAndNoteworthyPlugin.js

```diff
@@ -0,0 +1,195 @@
+        function nodeDisplay(node) {
+            return m('div', {'class': 'row node-styling noteworthy-spacing', onclick: function(){{ctrl.nodeDirect(node);}}},
+                m('div', {'class': 'col-sm-12'},
+                    m('h5', {'class': 'prevent-overflow', onmouseover: function(){ctrl.addToolTip(this);}}, m('em', ctrl.getTitle(node))),
+                    m('h5', {'class': 'prevent-overflow', onmouseover: function(){ctrl.addToolTip(this);}},  ctrl.getDescription(node)),
+                    m('h5', {'class': 'prevent-overflow'}, m('span', {'class': 'f-w-xl'}, 'Contributors: '), m('span',  ctrl.getContributors(node)))
+                )
+            );
+        }
+
+        function populateNodesSmallScreen () {
+            var formattedNodes = [];
+            formattedNodes.push(
+                m('div', {'class': 'row sm-new-and-noteworthy'}, m('div', {'class':'col-sm-10 col-sm-offset-1'},
+                    m('div', {'class': 'col-sm-6'}, m('h4', 'New and Noteworthy')),
+                    m('div', {'class': 'col-sm-6'}, m('h4', 'Most Popular')))
+            ));
+            for (var i = 0; i <= ctrl.popularNodes().length - 1; i++) {
+                var newNoteworthy = ctrl.newAndNoteworthyNodes()[i];
+                var popular = ctrl.popularNodes()[i];
+                formattedNodes.push(m('div', {'class': 'row sm-new-and-noteworthy'},
+                    m('div', {'class': 'col-sm-10 col-sm-offset-1'},
+                        m('div', {'class': 'col-sm-6'}, nodeDisplay(newNoteworthy)),
+                        m('div', {'class': 'col-sm-6'}, nodeDisplay(popular))
+                    )
+                ));
+            }
+            return formattedNodes;
+        }
+
+        function newAndNoteworthyProjectsTemplateXSScreen () {
+            return ctrl.newAndNoteworthyNodes().map(function(node){
+                return nodeDisplay(node);
+            });
+        }
+
+        function popularProjectsTemplateXSScreen () {
+            return ctrl.popularNodes().map(function(node){
+                return nodeDisplay(node);
+            });
+        }
+
+        function findMoreProjectsButton () {
+            return m('button', {type:'button', 'class':'btn btn-default m-v-lg', onclick: function(){
+                ctrl.redirectToSearch();
+            }}, 'Find more projects with advanced search');
+        }
+
+        return m('div', {'class': 'container'}, [
+            m('div', {'class': 'row'},
+                m('div', {'class': 'col-sm-1'}),
+                m('div', {'class': 'col-sm-11'}, m('h3', 'Discover Public Projects'))),
+
+            m('div', {'class': 'row xs-new-and-noteworthy'},
+                m('div', {'class': 'col-sm-6 col-xs-12'}, m('h4', 'New and Noteworthy'), newAndNoteworthyProjectsTemplateXSScreen()),
+                m('div', {'class': 'col-sm-6 col-xs-12'}, m('h4', 'Most Popular'), popularProjectsTemplateXSScreen ())
+            ),
+            populateNodesSmallScreen(),
```

Comment:
I will fix by getting rid of the populateNodesSmallScreen method - I originally did this looking for a way to get the Popular and New/Noteworthy divs side by side, the same height. I thought that on a larger screen, looping through the lists and putting a new/noteworthy node right beside a popular node would make the divs the same height.  And then I needed a different view on an xs screen, because on resize, I would have a 'New and Noteworthy' div and then a 'Popular' div, alternating.

Then I learned that bootstrap grids don't align heights of columns.